### PR TITLE
Get involved contribution

### PR DESCRIPTION
**Pull request by michellejae**
Added a bit more content to the Get Involved / Contributors guide.

### CODE REVIEW

**Comment by austinlparker** [comment]
Thanks for this! You'll need to sign the CLA - you can do this at https://identity.linuxfoundation.org/projects/cncf. Thanks!

**Comment by michellejae** [comment]
Ok, got a little lost! Ha. I created an account as an individual via my github but not sure what to do next. Do I also need to fill out the organization contribution form? Sorry can't seem to find out how to sign the CLA as an individual.

File: content/get-involved.md [hunk 1]
```diff
@@ -4,3 +4,42 @@ Title: "Get Involved"
 
 Are you interested in participating in the OpenTelemetry community? Read on to learn more
 about how to get involved.
+
+# Special Interest Groups
+We organize the community into Special Interest Groups (SIGs) in order to improve our workflow and more easily manage a community project.
+
+[Cross-language Specification] (https://github.com/open-telemetry/community#cross-language-specification)
```
Comment (lucperkins) [inline content/get-involved.md]: These links are not proper Markdown. Remove the space between the bracket and the parenthesis.

**Comment by austinlparker** [comment]
You'd use https://identity.linuxfoundation.org/?destination=node/285/individual-signup to sign as an individual.

File: content/get-involved.md (hunk 1 above)
Comment (michellejae) [reply content/get-involved.md]: fixed, i think! sorry about that.

**Comment by michellejae** [comment]
rad, that's how i signed up austin. sorry for all the back and forth.

**Comment by austinlparker** [comment]
Can you try making another commit (just add a space somewhere or new line or something) to get the CLA check to re-run?

**Comment by austinlparker** [comment]
Hm, still not working it seems... maybe try opening a new PR in a new branch? Sorry, sometimes the CLA bot gets stuck :(

**Comment by michellejae** [comment]
okay. do i close out this pr then? not sure proper etiquette / procedure.

**Comment by austinlparker** [comment]
Closed by #54

**Pull request closed without merging**